Services/Concurrency/RatedCounter.cs

```diff
@@ -177,6 +177,23 @@ public long GetPause()
             return false;
         }
 
+        public double GetThroughputForMessages()
```

Comment:
please add summary comments for the functions

        /// <summary>
        /// 
        /// </summary>